csv-utils/userToCsv.js

```diff
@@ -0,0 +1,43 @@
+exports.userToCsvHeader = (users) => {
+  const convertedUsers = [];
+
+  for (let i = 0; i < users.length; i += 1) {
+    const user = users[i];
+    const { Username, Attributes, UserLastModifiedDate } = user;
+    console.log(user);
```

Comment:
Why log this user here? If it is intentional, I would suggest filling the logged statement in with a bit more information.

Reply:
Removed console.log